### DISCUSSION TITLE
opening with VsCode

### DISCUSSION THREAD

**Discussion by Samthesurf** [sxyazi/yazi#271] [1. Q&A]
I configured yazi so that Neovim opens all text files, but how do I make it such that VsCode opens ipynb files

I tried this
[open]
rules = [
				{name ="*. ipynb", use = "code"} 
]
But this approach doesn't work, how do I go about it?

**Comment by sxyazi** [comment]
> {name ="*. ipynb", use = "code"}

your name contains whitespace, remove it and try again.

**Comment by swelborn** [comment]
i tried this, but doesn't seem to be working. do i need to set both an [opener] and [open] in my toml file? Is there anyway to debug what is happening in yazi on the backend?
